generators/client/templates/angular/src/main/webapp/app/_app.module.ts

```diff
@@ -1,3 +1,4 @@
+import './vendor.ts';
```

Comment:
No, webpack seems to be smart enough to not import it twice. Btw I added a visualizer for our bundles. When you build for prod, look at the "stats.html" file in target/www its really great !